acceptancetests/jujupy/k8s_provider/kubernetes_core.py

```diff
@@ -0,0 +1,332 @@
+# This file is part of JujuPy, a library for driving the Juju CLI.
+# Copyright 2013-2019 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the Lesser GNU General Public License version 3, as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the Lesser
+# GNU General Public License for more details.
+#
+# You should have received a copy of the Lesser GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Functionality for handling installed or other juju binaries
+# (including paths etc.)
+
+
+from __future__ import print_function
+
+import logging
+import tempfile
+import subprocess
+
+from .base import (
+    Base,
+    K8sProviderType,
+)
+from .factory import register_provider
+
+
+logger = logging.getLogger(__name__)
+
+
+LXD_PROFILE = """
+name: juju-{model_name}
+config:
+  boot.autostart: "true"
+  linux.kernel_modules: ip_tables,ip6_tables,netlink_diag,nf_nat,overlay
+  raw.lxc: |
+    lxc.apparmor.profile=unconfined
+    lxc.mount.auto=proc:rw sys:rw
+    lxc.cap.drop=
+  security.nesting: "true"
+  security.privileged: "true"
+description: ""
+devices:
+  aadisable:
+    path: /sys/module/nf_conntrack/parameters/hashsize
+    source: /dev/null
+    type: disk
+  aadisable1:
+    path: /sys/module/apparmor/parameters/enabled
+    source: /dev/null
+    type: disk
+"""
+
+BUNDLE = """
```

Comment:
We should deploy this from the charm store rather than embedding here.

Reply:
changed to deploy stable bundle from charmstore now.